src/main/resources/scripts/rwa-2829_mi_data_fixes.sql

```diff
@@ -0,0 +1,216 @@
+-- PRE - IMPLEMENTATION CHECKS
+-------------------------------------------
+select count(*)  from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS');
+-- 511,502
+
+select count(*), jurisdiction  from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') group by jurisdiction
+-- 14765	CIVIL
+-- 495899	IA
+-- 838	  PRIVATELAW
+
+select count(*) from reportable_task rt  where task_id in (select task_id from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS'))
+-- 88,471 (tasks to be marked)
+
+select count(*), state from reportable_task rt  where task_id in (select task_id from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS'))  group by state
+-- 426	  ASSIGNED    (tasks to be marked)
+-- 20   	COMPLETED   (tasks to be marked)
+-- 87053	TERMINATED  (tasks to be marked)
+-- 972	  UNASSIGNED  (tasks to be marked)
```

Comment:
Are we only fixing Terminated ones or all of them?  I though the problem only existed with the Terminated tasks.  Until then they all worked ok.  The ones that are incomplete will not get any new data anyway

Reply:
I am guessing it should still work but as above shows you'd be fixing more things than may be needed.